integration-libs/epd-visualization/root/config/epd-visualization-config.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { Config } from '@spartacus/core';
+import { UsageId } from '../models/usage-ids/usage-id';
+import { UsageIdDefinition } from '../models/usage-ids/usage-id-definition';
+
+@Injectable({
+  providedIn: 'root',
+  useExisting: Config,
+})
+export abstract class EpdVisualizationConfig implements Config {
+  public ui5?: Ui5Config;
+  public apis?: VisualizationApiConfig;
+  public usageIds?: UsageIdConfig;
+  public visualPicking?: VisualPickingConfig;
```

Comment:
Now those 4 keys are placed in the root level of the Spartacus config. Let's nest them in one common key that is specific to your feature library, e.g.:
```
  epdVisualisation: {
    ui5?: Ui5Config,
    apis?: VisualizationApiConfig,
    usageIds?: UsageIdConfig,
    visualPicking?: VisualPickingConfig,
 }
```

Reply:
I'm happy to do that.